common/config/rush/pnpm-lock.yaml

```diff
@@ -5177,6 +5177,15 @@ packages:
       webpack-cli: 4.7.2_webpack@5.38.1
     dev: false
 
+  /@welldone-software/why-did-you-render/6.2.1_react@16.14.0:
```

Comment:
You probably want to revert this? Or commit also the changes that enable it in the playground?